frontend/micro-ui/web/micro-ui-internals/packages/modules/hrms/src/components/EmployeeModuleCard.js

```diff
@@ -47,7 +47,7 @@ const EmployeeModuleCard = ({
           <div className="links-wrapper" style={{ width: "80%" }}>
             {links.map(({ count, label, link }, index) => (
               <span className="link" key={index}>
-                {link && link?.includes("https") ? (
+                {link && link?.includes("https") && link?.includes("ifix") ? (

```

Comment:
invalid logic @rakesh-wt-egov , this will again wont redirect to workbench , remove that ifix check condition